appdev/guides/writeable-dirs.rst

```diff
@@ -0,0 +1,90 @@
+Writable directories
+====================
+
+App confinement is part of the Ubuntu Touch security concept. Data can be
+exchanged between apps only according to the AppArmor policies. This being said,
+apps can only read and write files that are located in one of three app 
+specific directories explained in this guide.
+
+.. Note::
+    The app can write debug messages into the app log file located at 
+    ``/home/phablet/.cache/upstart/application-click-<fullappname>_<appname>_<version>.log``.
+    To write into the log file, use the Qt debug functions.
```

Comment:
I'd recommend adding a link to the debug functions you're talking about

Reply:
done